mantidimaging/core/operations/rescale/rescale.py

```diff
@@ -19,9 +19,7 @@ class RescaleFilter(BaseFilter):
 
     Intended to be used on: Any
 
-    When: Can be used to crop-out value regions of interest
-
-    When: Automatically used when saving images to uint16
+    When: Automatically used when saving images to unsigned int
```

Comment:
Might be better to say:
When: Automatically used when saving as 16 bit images

Reply:
Changed to "Automatically used when saving as unsigned integer image formats, to avoid clipping negative values".

Its the unsigned bit that matters.

Reply:
I thought image formats are always unsigned?

Reply:
Pretty sure that FITS and TIFF can be signed (though we currently only write uint16). Lots of graphics software wont open non uint tiffs.